Library/Formula/guile.rb

```diff
@@ -13,6 +13,14 @@ class Guile < Formula
     sha256 "51f5f379e25fab5cf8fb7cede02841aa716c0e90356705be2abc6a18c6af5371" => :mountain_lion
   end
 
+  devel do
+    url "http://git.savannah.gnu.org/r/guile.git", :tag => "v2.1.2",
```

Comment:
Sure thing! Is that something that `brew style`/ Rubocop should know to look for?